src/dune_rules/pkg_rules.ml

```diff
@@ -195,6 +186,48 @@ module Install_cookie = struct
     | Some f -> f
 end
 
+module Env_update = struct
+  include Dune_lang.Action.Env_update
+
+  let update kind ~new_v ~old_v ~f =
+    if new_v = "" then old_v
+    else
+      match kind with
+      | `Colon ->
+        let old_v = Option.value ~default:[] old_v in
+        Some (f ~old_v ~new_v)
+      | `Plus -> (
+        match old_v with
+        | None | Some [] -> Some [ Value.String new_v ]
+        | Some old_v -> Some (f ~old_v ~new_v))
+
+  let append = update ~f:(fun ~old_v ~new_v -> old_v @ [ Value.String new_v ])
+
+  let prepend = update ~f:(fun ~old_v ~new_v -> Value.String new_v :: old_v)
+
+  let set env { op; var = k; value = new_v } =
+    Env.Map.update env k ~f:(fun old_v ->
+        let append = append ~new_v ~old_v in
+        let prepend = prepend ~new_v ~old_v in
+        match op with
+        | Eq ->
+          if new_v = "" then if Sys.win32 then None else Some [ String "" ]
+          else Some [ Value.String new_v ]
+        | PlusEq -> prepend `Plus
+        | ColonEq -> prepend `Colon
+        | EqPlus -> append `Plus
+        | EqColon -> append `Colon
+        | EqPlusEq ->
+          (* TODO nobody uses this AFAIK *)
```

Comment:
Maybe remove this variant from `op` and display an error when parsing? After all it can always be added later.

Reply:
I do plan to implement this before the feature is ready. So it's fine to leave this for now. You or @gridbugs are welcome to beat me to it though.